sdk/servicebus/service-bus/.prettierrc.json

```diff
@@ -1,6 +1,9 @@
 {
-  "tabWidth": 2,
-  "singleQuote": false,
   "arrowParens": "always",
-  "printWidth": 100
+  "bracketSpacing": true,
```

Comment:
If we think this is a good prettier config for all of our projects, should we store it in the root of the repo and symlink it into each package's directory?

Reply:
done